src/db_queries/expenses.ts

```diff
@@ -8,23 +8,23 @@ const helpers = new Helpers();
 
 export class Expenses {
```

Comment:
Let's name these files `centers_expenses.ts` and `centers_expenses_paid.ts`.

Could you also use tab size 2 instead of 4 in all files?

Reply:
The project is missing prettier config file and vscode settings file that configures formatter settings. I have added both. Please check and this can be updated if needed.